requirements/edx/base.txt

```diff
@@ -99,7 +99,10 @@ django-memcached-hashring==0.1.2
 python-openid==2.2.5
 python-dateutil==2.1
 social-auth-app-django==1.2.0
-social-auth-core==1.6.0
+# Replaced by RG fork in github.txt
+# social-auth-core==1.6.0
+# Required for auth0 provider:
+python-jose==3.0.1
```

Comment:
Why do we need this package?
(UPDATE) It should be added to the requirements in the `social-core` package.

Reply:
Deleted, will be added to `social-core`